source/_integrations/netgear.markdown

```diff
@@ -17,15 +18,41 @@ ha_codeowners:
 ha_ssdp: true
 ---
 
-This platform allows you to detect presence by looking at connected devices to a [NETGEAR](https://www.netgear.com/) device.
+This platform allows you to detect presence by looking at connected devices to a [NETGEAR](https://www.netgear.com/) device and control the NETGEAR device.
+Both routers and acces points can be used with this integration, some acces points will not be automatically discoverd and need to be set up manually.
 
 {% include integrations/config_flow.md %}
 
-Most NETGEAR routers use port 5000 to communicate, however the following list of models are known to use port 80:
-- Nighthawk RAX50
-- Nighthawk X4S - AC2600 (R7800)
-- Orbi
-- XR500
-When setup through ssdp discovery the port schould be automatically detected.
+## Optional settings
+The options flow of the NETGEAR integration allows you to tweak the behavior of the NETGEAR integration and can be accesed as followes: in the sidebar of your Home Assistant instance click on "Configuration" -> "Integrations" -> find the NETGEAR integration and click "Configure".
```

Comment:
Maybe just remove this whole section? The location of an options flow is the same for all integrations that support that.
```suggestion
The options flow of the NETGEAR integration allows you to tweak the behavior of the NETGEAR integration and can be accessed as follows: in the sidebar of your Home Assistant instance click on "Configuration" -> "Integrations" -> find the NETGEAR integration and click "Configure".
```

Reply:
Sure I think you are right, I will remove it.

Reply:
done